web/client/src/library/components/editor/Editor.tsx

```diff
@@ -92,49 +93,61 @@ export default function Editor(): JSX.Element {
 
   function getSizesCodeEditorAndInspector(): [number, number] {
     const showInspector =
+      tab != null &&
       (tab.file.isSQLMeshModel || tab.file.isLocal) &&
       isFalse(isStringEmptyOrNil(tab.file.content))
 
     return showInspector ? [75, 25] : [100, 0]
   }
 
   return (
-    <SplitPane
-      sizes={sizesCodeEditorAndPreview}
-      direction="vertical"
-      minSize={0}
-      snapOffset={0}
-    >
-      <div className="flex flex-col overflow-hidden">
-        {isReadyEngine && (
-          <>
-            <EditorTabs />
-            <Divider />
-            <div className="flex flex-col h-full overflow-hidden">
-              <SplitPane
-                className="flex h-full"
-                sizes={sizesCodeEditorAndInspector}
-                minSize={0}
-                snapOffset={0}
-              >
-                <div className="flex flex-col h-full">
-                  <CodeEditor />
+    <div className="w-full h-full flex flex-col overflow-hidden">
+      <EditorTabs />
+      <Divider />
+      {tab == null ? (
+        <div className="flex justify-center items-center w-full h-full">
+          <div className="p-4 text-center text-theme-darker dark:text-theme-lighter">
+            <h2 className="text-3xl">Select File or Add New SQL Tab</h2>
```

Comment:
I don't feel strongly about this, but here's what I would say
```suggestion
            <h2 className="text-3xl">Select a file or add a new tab</h2>
```

Reply:
Should we rather rephrase as "create a new tab" or "open a new tab"? The word "add" here feels a bit weird to me.

Reply:
> add  a new tab
our new Tab has specific constraints , it can only be  a SQL file, and it is local and virtual (no saving option)

It is common pattern to capitalize words in call to action items 

![Screenshot 2023-04-10 at 11 57 50 AM](https://user-images.githubusercontent.com/5644753/230973612-b230b7e0-9206-41c5-bf42-e1a56e2019b0.jpg)

Reply:
> Should we rather rephrase as "create a new tab" or "open a new tab"? The word "add" here feels a bit weird to me.

On UI we are using "+" (addition ) , it is implying  that user be adding something. It is not a rule, just logic I'm following 

![Screenshot 2023-04-10 at 12 30 14 PM](https://user-images.githubusercontent.com/5644753/230981560-272cd474-765a-4a09-9df9-a49b93bbcc45.jpg)